### PR TITLE
fix(translations): add missing job types

### PR DESCRIPTION
**Pull request by ismay**
Fixes: 

- https://dhis2.atlassian.net/browse/DHIS2-15142
- https://dhis2.atlassian.net/browse/DHIS2-15094
- https://dhis2.atlassian.net/browse/DHIS2-15095
- https://dhis2.atlassian.net/browse/DHIS2-15096

### CODE REVIEW

File: src/services/server-translations/jobTypesMap.js [hunk 1]
```diff
@@ -1,27 +1,57 @@
 import i18n from '@dhis2/d2-i18n'
 
 const jobTypesMap = {
+    ACCOUNT_EXPIRY_ALERT: i18n.t('Account expiry alert'),
+    AGGREGATE_DATA_EXCHANGE: i18n.t('Exchange aggregate data'),
     ANALYTICS_TABLE: i18n.t('Analytics table'),
+    ANALYTICSTABLE_UPDATE: i18n.t('Analyticstable update'),
```
Comment (tomzemp) [inline src/services/server-translations/jobTypesMap.js:7]: `Analytics table`?

File: src/services/server-translations/jobTypesMap.js [hunk 2]
```diff
@@ -1,27 +1,57 @@
...
+    COMPLETE_DATA_SET_REGISTRATION_IMPORT: i18n.t(
+        'Complete data set registration import'
+    ),
     CONTINUOUS_ANALYTICS_TABLE: i18n.t('Continuous analytics table'),
     CREDENTIALS_EXPIRY_ALERT: i18n.t('Credentials expiry alert'),
     DATA_INTEGRITY: i18n.t('Data integrity'),
     DATA_SET_NOTIFICATION: i18n.t('Dataset notification'),
     DATA_STATISTICS: i18n.t('Data statistics'),
     DATA_SYNC: i18n.t('Data synchronization'),
+    DATAVALUE_IMPORT: i18n.t('Datavalue import'),
```
Comment (tomzemp) [inline src/services/server-translations/jobTypesMap.js:17]: `Data value`?

File: src/services/server-translations/jobTypesMap.js [hunk 3]
```diff
@@ -1,27 +1,57 @@
...
+    DATAVALUE_IMPORT_INTERNAL: i18n.t('Datavalue import internal'),
```
Comment (tomzemp) [inline src/services/server-translations/jobTypesMap.js:18]: `Data value`?

**Review by tomzemp — APPROVED** [review]
lgtm. I commented some word breaks which I wasn't sure were intentional or not

**Comment by ismay** [comment]
> lgtm. I commented some word breaks which I wasn't sure were intentional or not

Yeah I wasn't sure about those either. I just directly converted the snakecase to sentence case.

File: src/services/server-translations/jobTypesMap.js (hunk 1 above)
Comment (ismay) [reply src/services/server-translations/jobTypesMap.js]: [9dd992a](https://github.com/dhis2/scheduler-app/pull/493/commits/9dd992a6b4628c674bbdef9ae7c057690f2003fc)

File: src/services/server-translations/jobTypesMap.js (hunk 3 above)
Comment (ismay) [reply src/services/server-translations/jobTypesMap.js]: [9dd992a](https://github.com/dhis2/scheduler-app/pull/493/commits/9dd992a6b4628c674bbdef9ae7c057690f2003fc)

**Comment by ismay** [comment]
Fixed the word spacing as @tomzemp suggested, I'll wait for CI to pass and merge.

**Pull request merged**